README.md

```diff
@@ -1,24 +1,31 @@
-# Autolink Plugin (Beta) ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-autolink/master.svg)
+# Autolink Plugin ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-autolink/master.svg)
 
 This plugin creates regular expression (regexp) patterns that are reformatted into a Markdown link before the message is saved into the database.
 
 Use it to add custom auto-linking on your Mattermost system, such as adding links to your issue tracker based on the regexp patterns.
 
-**Supported Mattermost Server Versions: 5.2+ for 0.3+. 5.0 and 5.1 for all versions before 0.3**
+![image](https://user-images.githubusercontent.com/13119842/58675221-479ad680-8321-11e9-9ad1-b9c42238734f.png)
 
-## Installation
+*Posting a message containing a Jira issue key..*
 
-1. Go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-autolink) and download the latest release for your Mattermost server.
-2. Upload this file in the Mattermost **System Console > Plugins > Management** page to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Modify your `config.json` file to include the types of regexp patterns you wish to match, under the `PluginSettings`. See below for an example of what this should look like.
+![image](https://user-images.githubusercontent.com/13119842/58675165-11f5ed80-8321-11e9-9d41-91088a79a11b.png)
+
+*..automatically links to the corresponding issue in the Jira project*
+
+## Configuration
+
+1. Go to **System Console > Plugins > Management** and click **Enable** to enable the Autolink plugin.
+    - If you are running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-autolink), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+
+2. Modify your `config.json` file to include the types of regexp patterns you wish to match, under the `PluginSettings`. See below for an example of what this should look like.
 
 ## Usage
 
-Autolinks have 2 parts: a **Pattern** which is a regular expression search pattern utilizing the https://golang.org/pkg/regexp/ library, and a **Template** that gets expanded. You can create variables in the pattern with the syntax `(?P<name>...)` which will then be expanded by the corresponding template.
+Autolinks have 2 parts: a **Pattern** which is a regular expression search pattern utilizing the [Golang regexp library](https://golang.org/pkg/regexp/), and a **Template** that gets expanded. You can create variables in the pattern with the syntax `(?P<name>...)` which will then be expanded by the corresponding template.
```

Comment:
Please not that you can provided an optional `Scope` per link. See the first element of the example.

Reply:
A scope is a "Team Name[/Channel Name]", see https://github.com/mattermost/mattermost-plugin-autolink/blob/472c0ac69a15c9d7f6d75e48c85286972f442c4f/server/util.go#L9

Reply:
Oh nice, didn't realize this was supported. Opened a separate PR to clarify this in docs https://github.com/mattermost/mattermost-plugin-autolink/issues/57